src/charts/helpers/common.js

```diff
@@ -101,9 +101,10 @@ export function isNumeric (val) {
 export function extendIsValid (extent) {
   return extent
     && extent.length
-    && extent.filter(d => !isNaN(d.valueOf()) // valueOf also catches Invalid Date
```

Comment:
d could be null, so I moved this check after the null check

Reply:
neat trick with `valueOf()`! I suppose the `extent` array here will always have either Date or Numeric? If the extent had strings in it this would return false.